chart/Chart.yaml

```diff
@@ -76,8 +76,8 @@ annotations:
       links:
       - name: '#37197'
         url: https://github.com/apache/airflow/pull/37197
-    - description: 'Misc: Default airflow version to 2.8.3'
+    - description: 'Misc: Default airflow version to 2.9.1'
       kind: changed
       links:
-      - name: '#38036'
-        url: https://github.com/apache/airflow/pull/38036
+      - name: '#39433'
+        url: https://github.com/apache/airflow/pull/39433
```

Comment:
This will be done during the next release anyways, that's why it isn't done in those Airflow version bump PRs.

Said another way, this is just a different representation of the changelog for chart version 1.13.0.